docs/io/configuration/components/models/converters/arepo_to_tardis.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
<p>Do users have to generate the JSON themselves? Or can they get it from arepo? Seems like in either case the user would need the package?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1867/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Yes, the JSON files have to be generated by the user themselves. It is really only a shortcut to having the data available for the documentation, as getting it directly from the snapshot would introduce the arepo-snap-util package as a dependency. (This is not really advisable for two reasons: a) the package contains lots of unnecessary other utilties and the reelevant parts are mostly written in c anyway. b) It is as far as I know not publicly available.)

Rewriting the snapshot reader in Python is really not feasible; besides, this PR is written for the development version anyways, which is also not publicly available. And I recon that most people who use the development version also have the arepo-snap-util package. The public version should in principle also work, but I have no idea how these snapshots are read out. Here the users can save the data to JSON files and use the parser this way. But these files have to be generated by the user, because this is intended as a fallback for exotic Arepo snapshots anyway.